powerstore/resource_filesystem.go

```diff
@@ -263,6 +281,9 @@ func (r fileSystemResource) Schema(ctx context.Context, req resource.SchemaReque
 				Computed:            true,
 				Description:         "Type of filesystem: normal or snapshot.",
 				MarkdownDescription: "Type of filesystem: normal or snapshot.",
+				PlanModifiers: []planmodifier.Object{
+					objectplanmodifier.UseStateForUnknown(),
+				},
 				Attributes: map[string]schema.Attribute{
```

Comment:
Are we not supporting auto_lock, auto_delete and policy_interval as part of FLR attributes?

Reply:
Added